ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -472,6 +486,34 @@ async fn data_fetch(
                 step = 2.1,
                 "[Data Service] Data fetched from redis cache."
             );
+            LATEST_PROCESSED_VERSION
```

Comment:
could we add a step 2 before it starts and this can cover both of the fetching type. 

logically, we see this as step 2 _> step  2.1
                                                    |_> step 2.2

Reply:
we also have metric for step 3 which is sending to channel which covers both types of data fetches